contracts/assetsPool/AssetPoolUpgradeable.sol

```diff
@@ -88,40 +124,35 @@ contract AssetPoolUpgradeable is
     {
         require(amount > 0, "ZERO_AMOUNT");
```

Comment:
guys why don't you  try the costume  error handling i  think i have mention about it before it would be more better than this